docs/docs/associations.md

```diff
@@ -635,6 +635,104 @@ Series.hasOne(Video);
 Trainer.hasMany(Series);
 ```
 
+## Creating with associations
 
+New elements of an association can be created in one step, provided all elements are new.
```

Comment:
"New elements of an association" reads a little odd to me, perhaps "An instance can be created with nested association / dependencies in one step."